docs/content/preview/yugabyte-platform/security/create-kms-config/hashicorp-kms.md

```diff
@@ -125,6 +125,10 @@ You need to configure HashiCorp Vault in order to use it with YugabyteDB Anywher
 
     - `period` — If specified, the token can be infinitely renewed.
 
+    YBA automatically tries to renew the token every 12 hours after it has passed 70% of its expiry window; as a result, you should set the TTL or period to be greater than 12 hours.
```

Comment:
Can we also add the stuff about periodic tokens?

Reply:
@Sahith02 maybe - the docs already refer to period tokens being infinitely renewed. Thought this might be enough, with added link to Hashicorp docs. But if you think more detail needed, can add.